Text/TextAnalyticsForHealth/README.md

```diff
@@ -0,0 +1,118 @@
+If deploying through the ARM Template below, a Text Analytics Account will be created for you.
+
+## Settings
+
+This function requires `TEXT_ANALYTICS_API_ENDPOINT` and `TEXT_ANALYTICS_API_KEY` to be set in the settings.This is set automatically when deploying from the ARM Template.
+If running locally, this can be set in your project's debug environment variables (go to project properties, in the debug tab). This ensures your key won't be accidentally checked in with your code.
+If running in an Azure function, this can be set in the application settings.
+
+## Deployment
+
+[![Deploy to Azure](https://azuredeploy.net/deploybutton.svg)](https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure-Samples%2Fazure-search-power-skills%2Fmain%2FText%2FTextAnalyticsForHealth%2Fazuredeploy.json)
+
+## Sample Input:
+
+```json
+{
+    "values": [
+        {
+            "recordId": "foobar2",
+            "data":
+            {
+                "text":  "100mg Ibuprofin"
+            }
+        },
+        {
+            "recordId": "foo1",
+            "data":
+            {
+                "text":  "200mg Tylenol",
+                "languageCode":  "en"
+            }
+        }
+    ]
+}
+```
+The skill also accepts options which can be sent as headers in the request.
+
+| Name | Description | Default |
+| ---- | ----------- | ------- |
+| `timeout` | Allows you to set a timeout (in seconds) which will cause the skill to return early, so it doesn't timeout in the Cognitive Search Pipeline. | `230` |
+| `defaultLanguage` | Allows you to set a default language across all documents in case a language is not provided in the input | `"en"` |
+
+## Sample Output:
+
+```json
+{
+    "values": [
+        {
+            "recordId": "foobar2",
+            "data": {
+                "status": "succeeded",
+                "entities": [
+                    {...}
+                ],
+                "relations": [
+                    {...}
+                ]
+            },
+            "warnings": [],
+            "errors": []
+        },
+        {...}
+    ]
+}
+```
+
+## Sample Skillset Integration
+
+In order to use this skill in a cognitive search pipeline, you'll need to add a skill definition to your skillset. **If you deploy using the ARM template you can copy the URI from the output of the template.**
+Here's a sample skill definition for this example (inputs and outputs should be updated to reflect your particular scenario and skillset environment):
+
+```json
+{
+    "@odata.type": "#Microsoft.Skills.Custom.WebApiSkill",
+    "description": "Our new Bing entity search custom skill",
+    "context": "/document",
+    "uri": "[AzureFunctionEndpointUrl]/api/entity-search?code=[AzureFunctionDefaultHostKey]",
+    "batchSize": 1,
```

Comment:
Update description and uri (make sure the api path is the one for your function) and then also add the timeout and httpHeaders parameters (just use the default values that you use in your code for the purpose of the sample skill definition)

Reply:
Oh I did just realize httpHeaders is on line 116, I would remove it from there and move it up here and include the default settings for your special headers as well